velox/docs/functions/spark/comparison.rst

```diff
@@ -10,18 +10,28 @@ Comparison Functions
 
 .. spark:function:: equalnullsafe(x, y) -> boolean
 
-    Returns true if x is equal to y. Supports all scalar types. The
-    types of x and y must be the same. Unlike :spark:func:`equalto` returns true if both inputs
+    Returns true if ``x`` is equal to ``y``. Supports all types. The
+    types of ``x`` and ``y`` must be the same. Unlike :spark:func:`equalto` returns true if both inputs
     are NULL and false if one of the inputs is NULL.
     Corresponds to Spark's operator ``<=>``.
     Note that NaN in Spark is handled differently from standard floating point semantics.
     It is considered larger than any other numeric values. This rule is applied for functions
     "equalnullsafe", "equalto", "greaterthan", "greaterthanorequal", "lessthan", "lessthanorequal".
+    Nested nulls are allowed within input values if ``x`` is complex type. Nested 
```

Comment:
nit: since `equalnullsafe(null, null)` returns true instead of null, it seems not only nested nulls but also nulls at the top level are allowed, right?

Reply:
Yes, I reorganize the sentence.